ocs_ci/deployment/multicluster_deployment.py

```diff
@@ -175,3 +178,13 @@ def post_destroy_ops(self, cluster_list):
         """
         for cluster in cluster_list:
             cluster.post_destroy_ops()
+            prev = config.cur_index
+            config.switch_ctx(cluster.MULTICLUSTER["multicluster_index"])
+            ssl_key = config.DEPLOYMENT.get("ingress_ssl_key")
+            ssl_cert = config.DEPLOYMENT.get("ingress_ssl_cert")
+            if os.path.exists(ssl_key):
+                os.unlink(ssl_key)
+            if os.path.exists(ssl_cert):
+                os.unlink(ssl_cert)
+            ssl_certs.configure_custom_ingress_cert()
+            config.switch_ctx(prev)
```

Comment:
I'm little bit confused, why is the configuration of ingress certs (which I would expect will be performed post-deployment) in `post_destroy_ops`?

Reply:
oops, my mistake, instead of post_deploy I wrote it inside post_destroy. Changed the code properly.